### ISSUE TITLE
[CLOSE THIS] Request type colors

### ISSUE THREAD

**Issue by elizabeth-ux** [hackforla/311-data#1128]
The chartreuse single is indistinguishable from the mint green background on the safari iPhone 7. There's little difference between the homeless green and the single chartreuse. I would suggest using indigo instead. Or heliotrope or bright blue to distinguish it from the animal turquoise.

**Comment by EchoProject** [comment]
@melissahermes99 @elizabeth-ux is this issue still relevant? iPhone 7 seems to be pretty dated at this point, but I want to be sure our colors align with our design system.

**Comment by melissahermes99** [comment]
@EchoProject I just discussed this with Elizabeth. Apparently, this issue originated from user feedback after some H4LA community testing on the new map menu design. While the iPhone part of the issue is now becoming outdated, this issue also relates to the "single streetlight" color as a whole being difficult to distinguish from the "homeless encampment" green. I've run some of Elizabeth's suggested colors through a few sites to test for contrast and visual accessibility and I found that the bright blue would be the best to go with, specifically #8ACCFF. I am going to run this by Elizabeth once more before I implement the changes and inform dev that this will need to be updated. Or we can push this issue back until we start user testing. 
![image](https://user-images.githubusercontent.com/97048386/194796956-0f0f9cd4-d9e0-47b8-9d97-d71c003ff3e3.png)
